### PR TITLE
Updated gunmods and added modular selective fire [Icecoon, Part 3]

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
Part three of the Icecoon's Weapons Pack overhaul.

### CODE REVIEW

**Comment by BorkBorkGoesTheCode** [comment]
Work in progress

**Comment by BorkBorkGoesTheCode** [comment]
Ready.

File: data/mods/EW_Pack/ew_mods.json [hunk 1]
```diff
@@ -1,23 +1,44 @@
...
+        "install_time": 20000,
+        "loudness_modifier": 1,
+        "mode_modifier": [ [ "AUTO", "auto", 10 ] ],
+        "min_skills": [ [ "weapon", 3 ], [ "mechanics", 2 ] ]
+    },
+    {
         "type" : "GUNMOD",
-        "id" : "autofire_654",
-        "name" : "handmade auto-fire mechanism",
+        "id" : "imp_autofire_654",
```
Comment (mugling) [inline data/mods/EW_Pack/ew_mods.json]: Why the `id` change?

Comment (BorkBorkGoesTheCode) [inline data/mods/EW_Pack/ew_mods.json]: The gunmod was split into a player-craftable version, which cannot be removed, and a modular version. The original id is used for the modular version.

**Comment by Coolthulhu** [comment]
It isn't working correctly.

**Pull request closed without merging**